docs/linting/Troubleshooting.md

```diff
@@ -311,6 +311,69 @@ The following rules do not have equivalent checks in TypeScript, so we recommend
 - `import/no-unused-modules`
 - `import/no-deprecated`
 
+#### `import/extensions`
+
+There are two uses for this rule:
+
+1. to enforce file extensions are always used,
+2. to enforce file extensions are never used.
+
+##### Enforcing extensions are used
```

Comment:
Note that the font size for `#####`s is pretty small:

<img width="937" alt="Screenshot of headings with small sizes" src="https://user-images.githubusercontent.com/3335181/211625521-93053a88-b607-4d82-bdbd-54549c064c66.png">

Maybe a good time to bump that up a bit?

Reply:
How did you see this? Did you check it out locally?
When I tried the netlify deploy link above it doesn't include my changes :(

Reply:
Oh I tried this with `yarn start` locally. Weird that the deploy doesn't have it.